controller_manager/include/controller_manager/controller_manager.hpp

```diff
@@ -301,6 +304,9 @@ class ControllerManager : public rclcpp::Node
     const std::shared_ptr<controller_manager_msgs::srv::SetHardwareComponentState::Request> request,
     std::shared_ptr<controller_manager_msgs::srv::SetHardwareComponentState::Response> response);
 
+  CONTROLLER_MANAGER_PUBLIC
+  void init_resource_manager_cb(const std_msgs::msg::String & msg);
```

Comment:
We should call this differently and then call `init_resource_manager` method or other method later to update URDF

```suggestion
  void robot_description_callback(const std_msgs::msg::String & msg);
```

Reply:
renamed it